main.go

```diff
@@ -159,8 +159,16 @@ func main() {
 		lexer = lexers.Get(config.Input)
 	}
 
-	if config.Language != "" {
-		lexer = lexers.Get(config.Language)
+	if config.Language[0] != "" && len(config.Language) == 1 {
+		lexer = lexers.Get(config.Language[0])
+	} else if len(config.Language) == 2 {
+		language := lexers.Get(config.Language[0])
+		root := lexers.Get(config.Language[1])
+		lexer = chroma.DelegatingLexer(root, language)
+	}
+
+	if len(config.Language) > 2 {
+		printErrorFatal("languages can only be two", errors.New("--language can have only two inputs"))
 	}
```

Comment:
Why not do this first, to prevent unneeded comparisons, such that we return early if its not possible to continue.

Reply:
Oh right, I added this later so I forgot to remove it. Thanks!

Reply:
Should be fixed now!